feat:  Add information on the graphs in the List page

Lymah123 opened the pull request:
## Description

This PR adds the feature. 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Please use this format link issue numbers: Fixes #236 

## Mobile & Desktop Screenshots/Recordings

<!-- Visual changes require screenshots -->


## Steps to QA
<!-- 
Please provide some steps for the reviewer to test your change. If you have wrote tests, you can mention that here instead.

1. Click a link
2. Do this thing
3. Validate you see the thing working
-->


## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?



<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
  
  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

Lymah123 committed:
Add information on the graphs in the List feature

Lymah123 commented:
Hi, @BekahHW. Looking forward to the feedback.Thank you!

BekahHW commented on docs/features/lists.md:
```diff
@@ -30,4 +64,39 @@ There are three ways to add a list:
+## Activity
+
+The Activity tab gives you a graph view with more detailed information on each contributor, including the type of activity, repositories they've contributed to, and how they compare to each other.
+
```
There's also the Highlights tab that shows any Highlights on OpenSauced that have been made by the contributors on the list.

BekahHW replied:
I don't see the section on Highlights here.

BekahHW commented on docs/features/lists.md:
```diff
@@ -11,15 +11,49 @@ keywords:
 ![lists-page](../../static/img/lists-page.svg)
 
-## How to Create a List
+## Why Create Lists for Contributors and Maintainers?
```
For this section, use what's contained in the [Why Create a List section of the guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#why-create-a-list) for maintainers and [What You Can do With Lists](https://docs.opensauced.pizza/contributors/contributors-guide-to-open-sauced/#what-you-can-do-with-lists) for Contributors.

BekahHW commented on docs/features/lists.md:
```diff
@@ -11,15 +11,49 @@ keywords:
 
+Contributors can use Lists feature to track their own activity, while maintainers can use the feature to manage the community.
+
+## Lists Features for Contributors and Maintainers
```
I don't think we need this section if we have the Why Create Lists Section.

Lymah123 committed:
Changes made

Lymah123 commented:
Hi @BekahHW , I made changes based on the review. You can now check.

BekahHW commented on docs/features/lists.md:
```diff
@@ -11,15 +11,45 @@ keywords:
 ![lists-page](../../static/img/lists-page.svg)
 
-## How to Create a List
+## Why Create Lists for Contributors and Maintainers?
```
Let's cut the heading here and jump into two headings:

## Lists for Maintainers

Lists offer a variety of opportunities to track contributors for various reasons, including:

## Lists for Contributors

Lists offer contributors a way to gain insight into the world of open source contributions, including:

BekahHW commented on docs/features/lists.md:
```diff
@@ -11,15 +11,45 @@ keywords:
+
+- **VIP contributors**: Highlight and track at least five developers who are key maintainers or significant contributors.
+
+- **Sales leads**: Identify contributors whose engagement could potentially be turned into sales opportunities.
```
Let's cut the Sales Leads item and add this instead:

- **Your GitHub Teams**: Track the activity of your teams to identify if contributors are still active. For example, if you have members on your triage team, you can see their activity level on your repositories to determine if they're still active enough to participate in that role.

Lymah123 committed:
Adds update

Lymah123 commented:
Hi, @BekahHW! You can check.

adiati98 commented:
Hi @Lymah123,
We've merged some a PR before yours. Can you please resolve the merge conflicts here?

Thank you! 😊

Lymah123 committed:
resolce conflict

Lymah123 commented:
Hi @adiati98 .
Check the screenshot for the message on the conflicts.

My branch is up to date. 

Should I create a new PR?

Thank you!

![image](https://github.com/open-sauced/docs/assets/109049037/311cc39e-ebda-43f2-9409-f6ef5cedaccc)

Lymah123 committed:
Resolve conflict by removing lists.md

Lymah123 committed:
Adds lists-page

Lymah123 commented:
Hi @adiati98 , thank you. I read your article on it before even sending this. I was stuck still. But guess what, I have been able to fix it. Thank you for mentioning that I should start with upstream. I should blog on how I fixed it.

You can now check. Thank you so much!

adiati98 commented:
@Lymah123, good to hear that you've succeeded! I'll check it once I get back to my laptop 😊

---

If you already have upstream added as your remote repo, you can always pull the `main` branch directly from the upstream. 

This article is walking through beginners in updating their forked (origin) repo with the latest state of upstream before resolving conflicts because the good practice is to fork the upstream repo, clone the forked (origin) repo, and work on changes on this repo.

Adding steps to add upstream as remote repo and pull changes from upstream in the article will potentially confusinf and give them trouble when they accidentally work on the`main` branch.

Also, some repos don't allow changes/branches to be pushed directly to the upstream. For beginners, this will be too confusing and discouraging when they have to troubleshoot.

adiati98 committed:
Merge branch 'main' of https://github.com/open-sauced/docs into list-page-updates

adiati98 commented:
@BekahHW I just see this PR closely.

The "List" is now changed to "Contributor Insight". Do we still need this?

Lymah123 commented:
> @Lymah123, good to hear that you've succeeded! I'll check it once I get back to my laptop 😊
> 
> If you already have upstream added as your remote repo, you can always pull the `main` branch directly from the upstream.
> 
> This article is walking through beginners in updating their forked (origin) repo with the latest state of upstream before resolving conflicts because the good practice is to fork the upstream repo, clone the forked (origin) repo, and work on changes on this repo.
> 
> Adding steps to add upstream as remote repo and pull changes from upstream in the article will potentially confusinf and give them trouble when they accidentally work on the`main` branch.
> 
> Also, some repos don't allow changes/branches to be pushed directly to the upstream. For beginners, this will be too confusing and discouraging when they have to troubleshoot.

Okay,  thanks alot @adiati98 !

If I would be creating blog post on something like, that the article should simplifies the process for beginners and encourages a workflow that aligns with best practices in collaborative development.

Lymah123 commented:
> @BekahHW I only noticed merge conflicts before and just see this PR closely.
> 
> The "List" is now changed to "Contributor Insights". Do we still need this?

Hi @adiati98 , you mean there's no more lists section, it has changed to contribution insights?

adiati98 commented:
> Hi @adiati98 , you mean there's no more lists section, it has changed to contribution insights?

We just launched a new feature 3 days ago that included name changing.
Bekah has a day off. So, let's hear from her once she's back next week. 😊

Lymah123 commented:
> > Hi @adiati98 , you mean there's no more lists section, it has changed to contribution insights?
> 
> We just launched a new feature 3 days ago that included name changing. Bekah has a day off. So, let's hear from her once she's back next week. 😊

Alright! Thanks for the prompt responses, @adiati98 ❤ !

BekahHW commented:
Here's the new information. It's still the same feature, it just has a new name. https://github.com/open-sauced/docs/blob/main/docs/features/contributor-insights.md

Lymah123 commented:
Quick question:  I should change the lists in the file to contributors insights right? 

@BekahH and @adiati98 ! Happy new month to you both.

adiati98 commented:
> Quick question: I should change the lists in the file to contributors insights right?

We might do that. But let's hold this a bit longer and wait for @BekahHW. She will get back to you once she's back from her day off. 😊

Lymah123 commented:
Hi, @BekahHW and @adiati98 .

Any updates on the PR?

BekahHW commented:
Now that we shipped the updates with workspaces today, I'll be able to get
back to this tomorrow.

On Tue, Mar 12, 2024 at 12:24 PM Lymah123 ***@***.***> wrote:

> Hi, @BekahHW <https://github.com/BekahHW> and @adiati98
> <https://github.com/adiati98> .
>
> Any updates on the PR?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/open-sauced/docs/pull/258#issuecomment-1992064686>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIFZJRLI3HS3MYMIGJ5DDSTYX4T5NAVCNFSM6AAAAABDXDKLTOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSOJSGA3DINRYGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

BekahHW reviewed (changes requested):
Since we renamed things (lists are now contributor insights), you'll need to work in the new file. 

Here's the link to the docs currently: https://docs.opensauced.pizza/features/contributor-insights/

The file is `contributor-insights.md`.

Because we've added workspaces, take a bit of time to use them to create a contributor insight so you understand the flow.

When you link to an asset, it shouldn't be linked to the docs, it should be linked to the file in the repo. For example, you have this `![Here](https://docs.opensauced.pizza/assets/images/team-sync-09e7e2ab7b54fae335f5b4c40cce94b1.gif)` but it should be this: `![Team Sync gif](../../static/gif/team-sync.gif)`

We don't need to have the section on the highlights feature. If you think it's important to include, just link to the highlights feature file. 

Just take a look at [this section](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) in the maintainers guide. Stop once you hit the highlights section.

Lymah123 commented:
> Since we renamed things (lists are now contributor insights), you'll need to work in the new file.
> 
> Here's the link to the docs currently: https://docs.opensauced.pizza/features/contributor-insights/
> 
> The file is `contributor-insights.md`.
> 
> Because we've added workspaces, take a bit of time to use them to create a contributor insight so you understand the flow.
> 
> When you link to an asset, it shouldn't be linked to the docs, it should be linked to the file in the repo. For example, you have this `![Here](https://docs.opensauced.pizza/assets/images/team-sync-09e7e2ab7b54fae335f5b4c40cce94b1.gif)` but it should be this: `![Team Sync gif](../../static/gif/team-sync.gif)`
> 
> We don't need to have the section on the highlights feature. If you think it's important to include, just link to the highlights feature file.
> 
> Just take a look at [this section](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) in the maintainers guide. Stop once you hit the highlights section.

Hi @BekahHW , I got that. I will let you know if I have any questions. Thank you!

Lymah123 committed:
Adds contributor insights

Lymah123 committed:
Merge branch 'list-page-updates' of github.com:Lymah123/docs into list-page-updates

Lymah123 committed:
Merge branch 'main' into list-page-updates

Lymah123 commented:
Hi @BekahHW , and  @adiati98. I will be awaiting your feedback.

BekahHW reviewed (changes requested):
@Lymah123 go back to the original issue. Here are the tips again. Pay particular attention to the bolded ones. You should be writing general documentation and adding the correct images.

## Tips
- Use the list feature so you have an understanding of what you're looking at.
- If you have questions about the feature, leave them in the comments of this issue
- **Adding docs to this section should be targeted at a general audience. That means you can't take the information from the guide word for word. Think about maintainers, contributors, and organizations.**
- **Use the images in the guide, rather than creating new ones.**

Lymah123 commented on docs/features/lists.md:
@BekahHW , I deleted the file I have on my local machine and I cloned before working on it.

Should I close this PR and open a new one?

Lymah123 committed:
Delete docs/features/lists.md

Lymah123 commented:
@BekahHW, I just removed the list.md file from this PR. Let me know what other things to work on. Also, I only made changes to the previous submission before the release of the new feature.

Lymah123 committed:
Update contributor-insights.md

BekahHW commented on docs/features/contributor-insights.md:
```diff
@@ -17,22 +17,62 @@ Monitor individual commit histories, track who is actively engaged, identify alu
 
 Once your Contributor Insight Page is created, you'll be able to see an overview of all the contributors in your list, a graph of their activity, and a detailed breakdown of the contributors.
 
-:::tip
```
Why are you removing the tip here?

Lymah123 replied:
> Why are you removing the tip here?

I did not see a tip in the modified file. I saw the tip while I was trying to fix the merge conflict. A reason I removed it.

BekahHW replied:
Ah, I like the tip because it makes it stand out.

BekahHW commented on docs/features/contributor-insights.md:
```diff
@@ -17,22 +17,62 @@ Monitor individual commit histories, track who is actively engaged, identify alu
+- **Alumni contributors**: Keep an eye on contributors who have decreased their engagement or left the project.
+
 
 After logging in, navigate to "Insights" in the sidebar. Click the "+" next to it to be presented with options, and select "New Contributor Insight."
```
This line shouldn't be in between these lists.

Lymah123 replied:
> This line shouldn't be in between these lists.

@BekahHW ,  these are not in the files I have. I think something is not clear. I will just create a new branch and submit the changes I made.

BekahHW commented on docs/features/contributor-insights.md:
```diff
@@ -17,22 +17,62 @@ Monitor individual commit histories, track who is actively engaged, identify alu
+
+- **Track contributors for job opportunities**: If you're looking for a job, you can track the activity of the contributors you're interested in working with. This helps you understand the activity level of the repositories and the types of contributions being made, and you can even identify which company you want to contribute to.
+
+There are Three ways to add contributors to track:
```
This repeats information that you have below. Delete it here and keep the section below.

BekahHW commented on docs/features/contributor-insights.md:
```diff
@@ -17,22 +17,62 @@ Monitor individual commit histories, track who is actively engaged, identify alu
+
+## How to Create a Contributor Insight Page
+
+After logging in, navigate to your OpenSauced workspace. Click on the navigation bar of your workspace home page. Click on the plus sign beside insight. Two options will display, which are ; New repository insight and New contributor insight. Click on the New contributor insight. Then add the contributors by searching for their names or import contributors by syncing your GitHub following.
```
Use a bulleted list for directions

Lymah123 committed:
adds contributor insights

Lymah123 committed:
Merge branch 'list-page-updates' of github.com:Lymah123/docs into list-page-updates

Lymah123 commented:
@BekahHW , I just made changes based on your observation on my local machine. I would be waiting for the questions I left before submitting the update I made.

Lymah123 committed:
adds changes

Lymah123 commented:
Hi @BekahHW, you can check again.

BekahHW commented:
Hey @Lymah123, we're at our team onsite and a conference this week. I'll try to get to it, but may not until next week.

BekahHW commented:
@Lymah123, these are the headings you should have after the image:

```
## Why Create a Contributor Insight Page?
## How to Create a Contributor Insight Page 
## Using Your Contributor Insight Page
```

You can use the [maintainer guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) and the [Contributor Guide](https://docs.opensauced.pizza/contributors/contributors-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) to help you generate the content. 

We want a condensed version on this page, so summarize what those guides say for a general audience but link to those sections.

Lymah123 commented:
> Hey @Lymah123, we're at our team onsite and a conference this week. I'll try to get to it, but may not until next week.

Have fun there @BekahHW.

Lymah123 commented:
> @Lymah123, these are the headings you should have after the image:
> 
> ```
> ## Why Create a Contributor Insight Page?
> ## How to Create a Contributor Insight Page 
> ## Using Your Contributor Insight Page
> ```
> 
> You can use the [maintainer guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) and the [Contributor Guide](https://docs.opensauced.pizza/contributors/contributors-guide-to-open-sauced/#contributor-insights-connecting-with-contributors) to help you generate the content.
> 
> We want a condensed version on this page, so summarize what those guides say for a general audience but link to those sections.

Alright.  I will work on the feedback.  Thank you!

Lymah123 commented:
> @Lymah123 just a heads-up we just merged in a pretty big update to the docs, so you'll want to pull from main and install the new dependencies.


Noted @BekahHW. Thank you!

CBID2 commented:
Hey @Lymah123.You still working on this?
